README.md

```diff
@@ -1,14 +1,23 @@
-Backdrop Contributed Project Group Application
+What is the difference between Backdrop Contrib Project Group and backdrop-ops/contrib?
 ==============================================
 
-Interested in joining the growing Backdrop contributor community at
-https://github.com/backdrop-contrib?
+The Backdrop Contrib Project Group is an organization structure that retains ownership of the individual project repositories as developers come and go from projects.  Managing projects within the Backdrop Contrib Project Group facilitates the packaging and discovery of projects on Backdropcms.org, collaboration with other member of the Backdrop Contrib Project Group and security coverage. 
 
-To apply simply create a request as an issue at
-https://github.com/backdrop-ops/contrib/issues/new.
+The backdrop-ops/contrib/issues can be used to get feedback and find collaborators for a potential Backdrop project and to apply to join the growing community of developers maintaining a module or theme hosted at https://github.com/backdrop-contrib/ and discuss [META] issues realted to contributing to Backdrop.
 
-Please note: It is completely optional to post your modules, themes, or layouts
-in the Backdrop contrib group.
+Getting Started?
+==============================================
+
+The process of (porting a project from Drupal to Backdrop is well documented)[https://api.backdropcms.org/converting-from-drupal].
+
+If you intend to port a module from Drupal to Backdrop, simply create an issue titled "Intent to Port [project]" as an issue at https://github.com/backdrop-ops/contrib/issues/new
```

Comment:
"well documented" and "simply create" kind of speak to the "feeling small" coming from D7/D8. Removing those modifiers and keeping with the straight talk of how you do something I think would be better...e.g. what if I don't find opening the issue "simple"? Now I feel dumb...I was a psych major...